### ISSUE TITLE
[Bug]: Label sometimes is not updating

### ISSUE THREAD

**Issue by richxcame** [apertureless/vue-chartjs#808]
### Would you like to work on a fix?

- [ ] Check this if you would like to implement a PR, we are more than happy to help you go through the process.

## TypeError: Cannot read properties of undefined (reading 'destroy')

I use vue-i18n in labels. Sometimes, whenever i switch language labels aren't updating and in console shown like above.
Here is my code:
```
<template>
	<section>
		<Bar
			:chart-data="chartData"
			:chart-options="{ responsive: true, maintainAspectRatio: false }"
		/>
	</section>
</template>

<script>
import { mapState, mapActions } from 'vuex';
import 'chart.js/auto';
import { Bar } from 'vue-chartjs/legacy';

export default {
	data() {
		return {};
	},
	components: {
		Bar,
	},
	computed: {
		...mapState('admin', ['statistics']),
		chartData() {
			return {
				labels: [
					this.$t('january'),
					this.$t('february'),
					this.$t('march'),
					this.$t('april'),
					this.$t('may'),
					this.$t('june'),
					this.$t('july'),
					this.$t('august'),
					this.$t('september'),
					this.$t('october'),
					this.$t('november'),
					this.$t('december'),
				],
				datasets: [
					{
						label: this.$tc('order', 2),
						backgroundColor: '#388e3c',
						data: this.statistics,
					},
				],
			};
		},
	},
	methods: {
		...mapActions('admin', ['fetchStatistics']),
	},
	async created() {
		await this.fetchStatistics();
	},
};
</script>

```

### Reproduction

https://codesandbox.io/s/github/apertureless/vue-chartjs/tree/main/legacy/sandboxes/bar?file=/index.js

### chart.js version

v3.7.1

### vue-chartjs version

v4.0.5

### Possible solution

I don't have any idea

**Comment by aarbakkeinnovation** [comment]
I'm experiencing the same issue when updating a line chart with new data / labels.

I think it is related to the typo in this line:
https://github.com/apertureless/vue-chartjs/blob/ec82a9a4325f91ba9ec64301a1c076f54c5ce81e/legacy/src/Charts.js#L126

chartDestroy(_chartRef.currentt) should be changed to chartDestroy(_chartRef.current)

**Comment by thabarbados** [comment]
Hi, @richxcame  @aarbakkeinnovation, can you tell me if we fixed this bug in release 4.0.6?

**Comment by aarbakkeinnovation** [comment]
@thabarbados Its fixed in release 4.0.6 👍

**Comment by richxcame** [comment]
> Hi, @richxcame @aarbakkeinnovation, can you tell me if we fixed this bug in release 4.0.6?

Yes, it's fixed in 4.0.6. Thanx 🙌
